### PR TITLE
github: do not automatically merge on "extended-review" label

### PR DESCRIPTION
**Pull request by phlogistonjohn**
The "extended-review" label indicates that review is likely to take
an extended period of time and for bots (or even humans) to not automatically take
a PR.

This avoid the need to preemptively mark a PR as changes needed or
give a false impression with a "do-not-merge" label, which has a subtle
lack-of-quality implication.

### CODE REVIEW

**Review by agarwal-mudit — APPROVED** [review]

**Review by ansiwen — APPROVED** [review]
LGTM

**Comment by ansiwen** [comment]
Just out of curiosity: is it possible to use a label pattern like `extended-review*`, then we could add our names for more clarity like `extended-review-ansiwen`...

**Comment by phlogistonjohn** [comment]
> Just out of curiosity: is it possible to use a label pattern like `extended-review*`, then we could add our names for more clarity like `extended-review-ansiwen`...

I dunno, that'd be neat though. Could you please check out the mergify.io docs? It's not mutually exclusive with this so I figure we can just merge this (now that we have two reviews), and do another PR in the future if that is supported.

I'll merge this later today if I don't hear any NACKs to the above.

**Review by nixpanic — APPROVED** [review]

**Comment by nixpanic** [comment]
> Just out of curiosity: is it possible to use a label pattern like `extended-review*`, then we could add our names for more clarity like `extended-review-ansiwen`...

Is the [awaiting review from me](https://github.com/ceph/go-ceph/pulls/review-requested/@me) option not sufficient?

**Pull request merged**
